### PR TITLE
Update observability quickstart to include local instruction

### PR DESCRIPTION
**Pull request by tcnghia**
# Description

Update observability quickstart to include local instruction

## Issue reference

Please reference the issue this PR will close: #288 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] The quickstart code compiles correctly
* [ ] You've tested new builds of the quickstart if you changed quickstart code
* [x] You've updated the quickstart's README if necessary

### CODE REVIEW

File: observability/README.md [hunk 1]
```diff
@@ -8,7 +8,42 @@ In this quickstart you will:
 - Instrument an application for tracing and then deploy it.
 - Troubleshoot a performance issue.
 
-## Prerequisites
+## Configure self hosted mode
+For self hosted mode, on running `dapr init`:
```
Comment (mchmarny) [inline observability/README.md:12]: This is little unclear. Does this require `dapr init` first?

File: observability/README.md [hunk 2]
```diff
@@ -8,7 +8,42 @@ In this quickstart you will:
...
+
+1. The following YAML file is created by default in `$HOME/dapr/config.yaml` (on Linux/Mac) or `%USERPROFILE%\dapr\config.yaml` (on Windows) and it is referenced by default on `dapr run` calls unless otherwise overridden `:
```
Comment (mchmarny) [inline observability/README.md:14]: nit, there is a trailing tik before `:`

File: observability/README.md [hunk 3]
```diff
@@ -8,7 +8,42 @@ In this quickstart you will:
...
+spec:
+  tracing:
+    samplingRate: "1"
+    zipkin:
+	endpointAddress: "http://localhost:9411/api/v2/spans"
+```
+
+2. The [openzipkin/zipkin](https://hub.docker.com/r/openzipkin/zipkin/) docker container is launched.
+
+3. The applications launched with `dapr run` will by default reference the config file in `$HOME/dapr/config.yaml` or `%USERPROFILE%\dapr\config.yaml` and can be overridden with the Dapr CLI using the `--config` param. For example, the following command will launch a Node js app using the default config.yaml:
```
Comment (mchmarny) [inline observability/README.md:33]: nit ` Node.js`

File: observability/README.md [hunk 4]
```diff
@@ -8,7 +8,42 @@ In this quickstart you will:
...
+
+```bash
+dapr run --app-id mynode --app-port 3000 node app.js
+```
+
+### Viewing Traces
+As mentioned, tracing is set up out of the box when running `dapr init`. To
```
Comment (mchmarny) [inline observability/README.md:40]: Remove "As mentioned".

**Review by mchmarny — CHANGES_REQUESTED** [review]
Few minor changes

File: observability/README.md (hunk 1 above)
Comment (tcnghia) [reply observability/README.md]: I reworded.

File: observability/README.md (hunk 2 above)
Comment (tcnghia) [reply observability/README.md]: Removed. Thanks.

File: observability/README.md (hunk 3 above)
Comment (tcnghia) [reply observability/README.md]: Fixed.

File: observability/README.md (hunk 4 above)
Comment (tcnghia) [reply observability/README.md]: Removed.

**Comment by tcnghia** [comment]
> Few minor changes

Thanks a lot. I've addressed them all.

**Review by mchmarny — APPROVED** [review]
/lgmt

**Pull request merged**